tools/osx/BUILD

```diff
@@ -17,33 +17,20 @@ DARWIN_XCODE_LOCATOR_COMPILE_COMMAND = """
 genrule(
     name = "xcode-locator-genrule",
     srcs = select({
-        ":darwin": ["xcode_locator.m"],
-        ":darwin_x86_64": ["xcode_locator.m"],
+        "//src/conditions:darwin": ["xcode_locator.m"],
+        "//src/conditions:darwin_x86_64": ["xcode_locator.m"],
         "//conditions:default": ["xcode_locator_stub.sh"],
     }),
     outs = ["xcode-locator"],
     cmd = select({
-        ":darwin": DARWIN_XCODE_LOCATOR_COMPILE_COMMAND,
-        ":darwin_x86_64": DARWIN_XCODE_LOCATOR_COMPILE_COMMAND,
+        "//src/conditions:darwin": DARWIN_XCODE_LOCATOR_COMPILE_COMMAND,
+        "//src/conditions:darwin_x86_64": DARWIN_XCODE_LOCATOR_COMPILE_COMMAND,
         "//conditions:default": "cp $< $@",
     }),
     local = 1,
     output_to_bindir = 1,
 )
 
-# TODO(cparsons): Consolidate with config_settings under //src
-config_setting(
```

Comment:
Do you think we should instead move the ones from src here? Otherwise I can just drop this change. I didn't realize we were expecting folks to rely on this

Reply:
With some further investigation, it looks like the ones under `src/conditions` can also be referenced via `@bazel_tools//src/conditions:darwin`

I thus think we can indeed turn down one of these copies -- but either way it's going to be a breaking change for some users... we'll need to message it as such.
I think it might make more sense to turn down //src/conditions and keep the tools/osx copy.

Reply:
Updated only keeping the tools/osx ones

Reply:
I think I got all of them but we'll see what the tests say